### PR TITLE
Enable sync from google registry.

### PR DESCRIPTION
**Pull request by ipanova**
closes #3573
https://pulp.plan.io/issues/3573

### CODE REVIEW

**Comment by ipanova** [comment]
ok test

File: plugins/pulp_docker/plugins/auth_util.py [hunk 1]
```diff
@@ -46,7 +46,7 @@ def update_basic_auth_header(headers, username, password):
     return headers
 
 
-def request_token(downloader, request, auth_header):
+def request_token(downloader, request, auth_header, repo_name):
```
Comment (asmacdo) [inline plugins/pulp_docker/plugins/auth_util.py]: I am thinking that `repo_name` should default to `None` for calls that don't involve a repository, like GET /v2/

Comment (ipanova) [reply plugins/pulp_docker/plugins/auth_util.py]: agreed but for that we'd need to parse the path here https://github.com/ipanova/pulp_docker/blob/3a6d9a92466b8c6fab447fb1c8b6b16b8bec78f1/plugins/pulp_docker/plugins/registry.py#L481

also here we would need to massage the data to somehow figure out wher eto retrieve the path from so we can parse it https://github.com/ipanova/pulp_docker/blob/3a6d9a92466b8c6fab447fb1c8b6b16b8bec78f1/plugins/pulp_docker/plugins/importers/sync.py#L433

imo it does not worth. thanks for lgtm merging.

**Comment by dkliban** [comment]
ok test

**Comment by ipanova** [comment]
@dkliban the builder is again not seeing last changes done to the test cases, i merged because tests are passing locally.

**Pull request merged**